### PR TITLE
Inserting data

### LINKED ISSUE #32: US 4a - Pull Assets Data

### PR DESCRIPTION
**Pull request by DanielMulitauopele**
# Pivotal URL or Waffle Card number:

closes #32 

# What does this PR do?

Pulls in assets data from endpoint and styles the remainder of the page

- [] Wrote Tests
- [X] Implemented
- [] Reviewed

# Neccesary checkmarks:
- [] All Tests are Passing

- [X] The code will run locally

## Type of change
- [X] New feature
- [] Bug Fix

# Check the correct boxes

- [] This broke nothing
- [X] This broke some stuff
- [] This broke everything

# Testing Changes
- [X] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [X] My code has no unused/commented out code
- [X] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have fully tested my code

# Please Include a link to a gif of your feelings about this branch

### CODE REVIEW

**Comment by DanielMulitauopele** [comment]
@ameseee

File: assets.html [hunk 1]
```diff
@@ -4,18 +4,21 @@
...
-
-
-
+    <div class="coin-header">
+      <p class="rank">Rank</p>
+      <p class="symbol">Symbol</p>
+      <p class="name">Name</p>
+      <p class="price">Price</p>
+    </div>
+    <div class="wrapper">
```
Comment (ameseee) [inline assets.html:21]: @DanielMulitauopele I would probably give this class a more semantic name, mainly for scaling. You'll have tons of wrappers in a medium sized page, you know.

File: lib/assets.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const getAssets = () => {
+  $('.listed-coin').html('');
+
+  fetch(`https://guarded-reef-25579.herokuapp.com/api/v1/assets`)
+    .then(response => response.json())
```
Comment (ameseee) [inline lib/assets.js:5]: @DanielMulitauopele all your JS looks really nice. We didn't get into this in class, but one thing about the `.catch` is it doesn't save us from 400s and 500s, we have to build out our own error handling. My go to resource for that is here: https://css-tricks.com/using-fetch/#article-header-id-5

File: lib/assets.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
+.wrapper {
```
Comment (ameseee) [inline lib/assets.css:1]: @DanielMulitauopele I don't actually think it's that much of a mess! I like the way you made the three files to separate the main, nav bar and login, that's really nice. Like we talked about with the mobile first stuff last week... I think that ideally we would rip out all of the elements with `position: fixed` and find a way to do without it - that will make moving it to desktop friendly a lot easier. But I'd rather you focus on your JS😊

**Pull request merged**